tests/e2e/api/controllers/changes.js

```diff
@@ -23,19 +24,12 @@ function assertChangeIds(changes) {
   expect(_.pluck(changes, 'id').sort()).toEqual(expectedIds.sort());
 }
 
-function requestChanges(username, ids, last_seq=0) {
+function requestChanges(username, params = {}) {
+  const queryParams = _.map(params, (value, key) => `${key}=${value}`).join('&');
```

Comment:
I think we have access to jquery here so consider using the [$.param](http://api.jquery.com/jquery.param/) function. We've used it elsewhere and it works well.

Reply:
If there is a way to access browser's jQuery, I couldn't find it. The `$` in Protractor is a shortcut for `element(by.css())`.